lib/ui/window.dart

```diff
@@ -867,35 +876,44 @@ enum Brightness {
   light,
 }
 
-/// The [SingletonFlutterWindow] representing the main window for applications
-/// where there is only one window, such as applications designed for
-/// single-display mobile devices.
+/// Deprecated. Will be removed in a future version of Flutter.
+///
+/// This property has been deprecated to prepare for Flutter's upcoming support
+/// for multiple views and eventually multiple windows.
 ///
-/// Applications that are designed to use more than one window should interact
-/// with the `WidgetsBinding.instance.platformDispatcher` instead.
+/// It used to represent the only window into which an application could draw
+/// content while giving the application access to window- and platform-specific
+/// functionality.
```

Comment:
Will the `window` singleton be removed before we support multi-windows? I slightly prefer the original sentence here, but I don't feel strongly about this:

```dart
/// The [SingletonFlutterWindow] representing the main window for applications
/// where there is only one window, such as applications designed for
/// single-display mobile devices.
```